package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "metrix-web-app",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@reduxjs/toolkit": "^1.7.1",
+    "@testing-library/jest-dom": "^5.16.1",
+    "@testing-library/react": "^12.1.2",
+    "@testing-library/user-event": "^7.2.1",
+    "axios": "^0.24.0",
+    "eslint-config-airbnb-base": "^15.0.0",
+    "eslint-plugin-markdown": "^2.2.1",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-helmet": "^6.1.0",
+    "react-redux": "^7.2.6",
+    "react-router-dom": "^6.2.1",
+    "react-scripts": "^2.1.3",
+    "react-test-renderer": "^17.0.2",
+    "redux": "^4.1.2",
+    "redux-logger": "^3.0.6",
+    "redux-thunk": "^2.4.1"
+  },
```

Comment:
- Please can't run the App. I think you have an issue with your package.json. Kindly check your setup
- I cross-check at my end. I did not work
![1](https://user-images.githubusercontent.com/63972670/197367845-bd6ac571-dce2-4ae0-ba7d-add88b16a439.JPG)